SECURITY.md

```diff
@@ -13,10 +13,12 @@ using the public PGP keys located below.
 
 |Maintainer	|Email	|PGP Key	|
 |---	|---	|---	|
+|Patrick Roy|roypat@amazon.co.uk|https://github.com/roypat.gpg|
+|Jonathan Woollett-Light|jcawl@amazon.co.uk|https://github.com/JonathanWoollett-Light.gpg|
 |Andreea Florescu |fandree@amazon.com |https://github.com/andreeaflorescu.gpg |
 |Laura Loghin |lauralg@amazon.com	|https://github.com/lauralt.gpg	|
 |Rob Bradford |robert.bradford@intel.com |https://github.com/rbradford.gpg |
```

Comment:
My email address has also changed - to rbradford@rivosinc.com

Reply:
Fixed.